libs/components/src/lib/constants/constants.ts

```diff
@@ -0,0 +1,20 @@
+export const prBarStrColor = '#021691';
+export const prBarTrailColor = '#B0C4DE';
+
+export const prBarProfileQ1Per = 20;
+export const profileQ1Form = 'profile_questions_1';
+export const profileQ1HR = 'hourly_rate';
+export const profileQ1Descr = 'description';
+export const profileQ1Pos = 'position';
+export const profileQ1AvTime = 'available_time';
+export const profileQ1Edu = 'education';
+export const profileQ1EduInfo = 'information-about-education1';
+export const profileQ1EduForm = 'education_from';
+export const profileQ1EduTo = 'education_to';
+export const profileQ1WorkHistoryWrapper = 'work_history_wrapper';
+export const profileQ1WorkHistory = 'work_history';
+export const profileQ1WorkFrom = 'work_from';
+export const profileQ1WorkTo = 'work_to';
+export const profileQ1SuffixUSDH = 'USD per hour';
+
+export const prBarProfileQ2Per = 50;
```

Comment:
hmmm maybe intead of 10 exports make one constant profileForm, and then in the component use something like profileForm.profileQ1HR, it will decrease amount of import variables